data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+		system "Wah Yoot"
+		fleet
+			names "hai"
+			variant
+				"Shield Beetle"
+	npc
+		government "Hai (Unfettered)"
+		personality mining uninterested launching
+		system "Wah Yoot"
+		fleet
+			names "hai"
+			variant
+				"Lightning Bug"
+	npc
+		government "Hai (Unfettered)"
+		personality harvests uninterested staying launching
+		system "Wah Yoot"
+		fleet
+			names "hai"
+			variant
+				"Water Bug"
+
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		log `Gathered materials for the Unfettered shipyards. They're building sizeable ships here.`
+		outfit "Silicon" -100
+		outfit "Silver" -15
+		payment 500000
+		dialog `You bring your cargo to the local Unfettered shipyards, and they hand you <payment>. You see huge ships being built in the distance.`
+
+
+
+outfit "Loot"
+	category "Special"
+	cost 1000
+	thumbnail "outfit/unknown"
+	"mass" 1
+	description "You're not sure what this is, but you feel your hair go up when you're near the cargo... Probably best not to touch it."
+
+ship "Behemoth" "Behemoth (Mammoth)"
+	outfits
+		"Boulder Reactor"
+		"Chameleon Anti-Missile"
+		"Fuel Pod"
+		"Hai Diamond Regenerator"
+		"Hai Fissure Batteries"
+		"Hai Tracker" 112
+		"Hai Tracker Pod" 2
+		"Pulse Turret" 5
+		"Scram Drive"
+		"Small Radar Jammer"
+		"X3200 Ion Steering"
+		"X3700 Ion Thruster"
+	cargo
+		outfits
+			"Loot" 400
+			"Gold Bars" 15
+
+mission "Unfettered: Loot: Ion Cannon Materials"
+	name `Loot cargo ship`
+	description `There is an undefended cargo ship near <waypoints>. Find it and bring its cargo back!`
+	source "Darkcloak"
+	to offer
+		has "Unfettered: Solifuge 5: Materials For Ships: done"
+		or
+			not "Unfettered: offered loot"
+			random < 30
+	on defer
+		set "Unfettered: offered loot"
+	on offer
+		conversation
+			`When passing in front of a bar, you notice Si-Ra. He seems to be having quite a lot of fun. The bar itself does not seem that alien to you.`
+			choice
+				`	(Join him.)`
+				`	(I don't have time for this now.)`
+					defer
+			`	The bar almost feels familiar. There are of course weird beverages, and the general architecture of it is simple to say the least, like all Unfettered buildings. But, in a way, it looks similar to pirate bars. Maybe some things are common between alien species...`
+			`	As you enter, you notice he's speaking in your language, probably to make fun of humans. What he says does not seem to have much sense to it - he's probably had one too many drinks. When he sees you, he starts shouting. "This is my friend, Captain <last>, folks! He doesn't mess around - the holes in my old ship can attest to that!"`
+			choice
+				`	"Old ship? You don't have it any more?"`
+					goto ship
+				`	"To be fair, you did ask for it."`
+			`	"True... true... I even provoked ya, didn't I?" He's probably not angry about it, but he really seems agitated - so much so that he spills some of the purple liquid from his drink onto the floor.`
```

Comment:
Agitated and angry seem too similar for you to be not angry but also agitated. Maybe there's a better word we could use here?

Reply:
Excited